ddtrace/contrib/grpc/aio_server_interceptor.py

```diff
@@ -0,0 +1,261 @@
+import inspect
+from typing import Any
+from typing import Awaitable
+from typing import Callable
+from typing import Dict
+from typing import Iterable
+from typing import Union
+
+import grpc
+from grpc import aio
+from grpc.aio._typing import RequestIterableType
+from grpc.aio._typing import RequestType
+from grpc.aio._typing import ResponseIterableType
+from grpc.aio._typing import ResponseType
+
+from ddtrace import Pin
+from ddtrace import Span
+from ddtrace import config
+from ddtrace.vendor import wrapt
+
+from .. import trace_utils
+from ...constants import ANALYTICS_SAMPLE_RATE_KEY
+from ...constants import ERROR_MSG
+from ...constants import ERROR_TYPE
+from ...constants import SPAN_MEASURED_KEY
+from ...ext import SpanTypes
+from ...internal.compat import to_unicode
+from ..grpc import constants
+from ..grpc.utils import set_grpc_method_meta
+
+
+Continuation = Callable[[grpc.HandlerCallDetails], Awaitable[grpc.RpcMethodHandler]]
+
+
+# Used to get a status code from integer
+# as `grpc._cython.cygrpc._ServicerContext.code()` returns an integer.
+_INT2CODE: Dict[int, grpc.StatusCode] = {s.value[0]: s for s in grpc.StatusCode}
```

Comment:
This is more a comment for @majorgreys 

I know this code only works on newer Python, but should we use a type comment here to be consistent?

I am not sure I have a strong opinion, just calling it out if we want to make a decision here or not.

Reply:
Maybe I shouldn't have added the type hint because in other codes type comments are used so that they can run with old Python.
Since the name of the variable `_INT2CODE` almost represents its type, no type hint would be fine.

Reply:
Deleted the type hint in 1d081c6.

Reply:
I actually think it was probably fine, since this is a Python 3 specific file we probably have typing available natively, but it is also ok that you removed it!